src/Microsoft.ML.OnnxTransformer/OnnxSequenceType.cs

```diff
@@ -94,6 +94,12 @@ public override int GetHashCode()
         /// </summary>
         public override void Register()
         {
+            // this happens when use OnnxSequenceType attribute without specify sequence type
+            if (_elemType == null)
+            {
+                throw new Exception("Please specify sequence type when use OnnxSequenceType Attribute.");
+            }
```

Comment:
please do not use braces for single line if condition

Reply:
Actually this is best practice to always use braces for if condition, in my previous team it is also suggested to always use braces for if condition to avoid further problem.

There is also some discussion online, please check out and see whether this make sense to you: [https://softwareengineering.stackexchange.com/questions/16528/single-statement-if-block-braces-or-no](https://softwareengineering.stackexchange.com/questions/16528/single-statement-if-block-braces-or-no) 



---
In reply to: [330344007](https://github.com/dotnet/machinelearning/pull/4272#discussion_r330344007) [](ancestors = 330344007)

Reply:
Offline discussed with Zeeshan, this can be debatable, I will just follow the coding style in this team, not a big issue

---
In reply to: [330672086](https://github.com/dotnet/machinelearning/pull/4272#discussion_r330672086) [](ancestors = 330672086,330344007)